dev/com.ibm.ws.jca_fat/test-resourceadapters/adapter.regr/src/com/ibm/adapter/XATerminatorWrapper.java

```diff
@@ -0,0 +1,299 @@
+/*******************************************************************************
+ * Copyright (c) 2003, 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.adapter;
+
+import javax.resource.ResourceException;
+import javax.resource.spi.XATerminator;
+import javax.transaction.xa.XAException;
+import javax.transaction.xa.Xid;
+
+import com.ibm.adapter.work.FVTWorkDispatcher;
+import com.ibm.ejs.ras.Tr;
+import com.ibm.ejs.ras.TraceComponent;
+
+/**
+ * @author swai
```

Comment:
Do we want to leave the `@author` in on the move to OL? Not sure if there were others, this one just caught my eye.

Reply:
Removed.